src/componentes/coffee-card.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import mocha_late from '../photo/mocha-late.png';
+import double_espresso from '../photo/double-espresso.png';
+
+class Brown_coffee_card extends Component {
+    state = {  }
+    render() { 
+        return ( 
+            <div className="coffee-item">
+                 <div className="small-image"><img src={mocha_late} alt="mocha late"></img></div>
+                <div className="small-image-text">
+                    <h4>Mocha Late</h4>
+                    <p className="gray-color">Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain Coffea species.</p>
+                </div>
+            </div>
+         );
+    }
+}
+
+class Blue_coffee_card extends Component {
```

Comment:
Please use Pascal case for the class and the file names convention 
see 
```suggestion
class BlueCoffeeCard extends Component {
```